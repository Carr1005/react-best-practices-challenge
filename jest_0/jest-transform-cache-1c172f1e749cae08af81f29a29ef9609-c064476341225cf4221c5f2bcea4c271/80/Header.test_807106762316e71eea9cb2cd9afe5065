'use strict';var _jsxFileName = '/var/tunnel2host/Vynca-Challenge/vynca-challenges/src/Header/Header.test.js';var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _Header = require('../Header');var _Header2 = _interopRequireDefault(_Header);


var _reactTestRenderer = require('react-test-renderer');var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}

test('Header test', () => {
  const component = _reactTestRenderer2.default.create(
  _react2.default.createElement(_Header2.default, { __source: { fileName: _jsxFileName, lineNumber: 9 }, __self: undefined }));

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});