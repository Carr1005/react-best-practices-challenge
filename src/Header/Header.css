i + strong {
	margin: 0px 0px 0px 8px;
}