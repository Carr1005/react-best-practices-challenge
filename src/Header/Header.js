import React, { Component } from 'react';
import './Header.css';

class Header extends Component {
  render() {
    return (
      <div id="top" className="navbar navbar-dark bg-dark box-shadow">
        <div className="container d-flex justify-content-between">
          <a href="#" className="navbar-brand d-flex align-items-center">
            <i className="material-icons">photo_camera</i>
            <strong>Album</strong>
          </a>
        </div>
      </div>
    );
  }
}

export default Header;
