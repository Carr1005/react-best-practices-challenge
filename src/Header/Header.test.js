import React from 'react';
import Header from './Header';


import renderer from 'react-test-renderer';

test('Header test', () => {
  const component = renderer.create(
    <Header></Header>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});