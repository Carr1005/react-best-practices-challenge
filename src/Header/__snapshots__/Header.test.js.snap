// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test 1`] = `
<div
  className="navbar navbar-dark bg-dark box-shadow"
  id="top"
>
  <div
    className="container d-flex justify-content-between"
  >
    <a
      className="navbar-brand d-flex align-items-center"
      href="#"
    >
      <i
        className="material-icons"
      >
        photo_camera
      </i>
      <strong>
        Album
      </strong>
    </a>
  </div>
</div>
`;
